# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ipconfig/compile-fix.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur ipconfig-orig/netdev.c ipconfig/netdev.c
--- ipconfig-orig/netdev.c	2006-06-29 17:14:38.000000000 +0000
+++ ipconfig/netdev.c	2006-07-25 20:39:37.000000000 +0000
@@ -11,6 +11,9 @@
 #include <string.h>
 #include <unistd.h>
 #include <net/if.h>
+#ifdef _NET_IF_H
+#define _LINUX_IF_H
+#endif
 #include <net/if_arp.h>
 #include <netinet/in.h>
 #include <linux/route.h>
